app/fastTrack/src/appRegistry/__tests__/__snapshots__/FastTrackBanner.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`FastTrackBanner Should render correctly 1`] = `null`;
```

Comment:
Is this the expected output of this component?

Reply:
Oh, definitely not. That component renders `PrivateApiRenderer` which uses `QueryRenderer`. I can't find any test that uses `QueryRenderer` or `PublicApiRenderer`. Can you give me some pointers, maybe I don't need to test this?

Reply:
So, actually, the test is not wrong, since 

```
// PrivateApiRenderer.js
 if (props.accessToken === null) {
    // TODO: Return login
    return null;
  }
```

You could do this: 

```
const component = renderer.create(
      <AuthContext.Provider accessToken="token">
        <FastTrackBanner bookingId={123} kwAuthToken="x.y.z" />,
      </AuthContext.Provider>,
    );
```

But please mock the `QueryRenderer` component, unless you will actually call the server.

Not sure what good that test would do though. I would rather consider just shallow rendering.